kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -1503,6 +1503,14 @@ object KyuubiConf {
       .checkValue(_ > 0, "the maximum must be positive integer.")
       .createWithDefault(10)
 
+  val SESSION_ENGINE_DESTROY_TIMEOUT: ConfigEntry[Long] =
+    buildConf("kyuubi.session.engine.destroy.timeout")
```

Comment:
"startup" should be mentioned, otherwise the name is misleading, destroy engine sounds too dangerous

```suggestion
    buildConf("kyuubi.session.engine.startup.destroy.timeout")
```

Reply:
ok

Reply:
ignored the comments?